### PR TITLE
Development

### PR DESCRIPTION
**Pull request by EdinTC**

### CODE REVIEW

File: src/App.svelte [hunk 1]
```diff
@@ -1,10 +1,12 @@
 <script>
   import { onMount } from 'svelte';
   import { getQueryParam, extractHostname, updateURL } from './utils';
+  import Header from "./Header.svelte";
   import Results from "./Results.svelte";
+  import Footer from "./Footer.svelte";
 
   export let promise;
-  const apiUrl = "https://api.checq.intercube.io";
+  const apiUrl = "https://api.checq.intercube.io/lookup";
```
Comment (Jensderond) [inline src/App.svelte:9]: Does this endpoint work exists on the master branch?

File: src/Branding.svelte [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<div class="branding whitespace-normal">
```
Comment (Jensderond) [inline src/Branding.svelte:1]: I don't believe this has been styled yet 😂

Comment (EdinTC) [reply src/Branding.svelte:1]: I removed the reference to this component in App.svelte.

File: src/App.svelte (hunk 1 above)
Comment (EdinTC) [reply src/App.svelte:9]: Good catch doesn't seem like it. It is something you added in the original pull request from task/improvements. Okay if I remove it for now?

**Comment by Jensderond** [comment]
LGTM

**Pull request merged**